src/main/java/GermaBot.java

```diff
@@ -0,0 +1,56 @@
+import java.util.Scanner;
+
+public class GermaBot {
+    public static int getIdx(String input) {
+        return Integer.parseInt(input.substring(input.indexOf(" ") + 1)) - 1;
+    }
+
+    public static void main(String[] args) {
+        String WelcomeMessage = "____________________ \n"
+                + "Hello! GermaBot here! \n"
+                + "What may I do for you this fine day? \n"
+                + "____________________";
+        System.out.println(WelcomeMessage);
+        Task[] toDoList = new Task[100];
+        int counter = 0;
+        while (true) {
+            String echo;
+            Scanner in = new Scanner(System.in);
+            echo = in.nextLine();
+            if (echo.equals("bye")) {
+                break;
+            }
+            if (echo.equals("list")) {
+                int printCounter = 1;
+                System.out.println("Gotcha! Here are your tasks:");
+                for (int j = 0; j < counter; j++) {
+                    if (toDoList[j] == null) {
+                        break;
+                    }
+                    System.out.println(printCounter + ". [" + toDoList[j].getStatusIcon() + "] " + toDoList[j].getDescription());
+                    printCounter++;
+                }
+            } else if (echo.contains("unmark")) {
+                int idx = getIdx(echo);
```

Comment:
Similar to the comment for getIndex. Perhaps can name "idx" to some other variable name which is more easy to follow when debugging.